spec/controllers/v1/client_details_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
This test doesn't do anything. Do we plan to write a controller test? If we want to keep this file to add tests to,  it should be renamed to applicants_controller_spec to be consistent with the name of the controller. If we don't want a test here then the file can be deleted.